app/qml/onboarding/MMLogin.qml

```diff
@@ -19,12 +19,15 @@ Page {
   width: parent.width
 
   signal backClicked
-  signal signInClicked
+  signal signInClicked ( string username, string password )
   signal signUpClicked
-  signal changeServerClicked
+  signal changeServerClicked ( string newServer )
   signal forgotPasswordClicked
 
+  property string warningMsg //TODO
+  property bool canSignUp: true //TODO
```

Comment:
For EE servers there is a flag `user_self_registration` to determine if we should show or hide registration option. We have ignored it previously but might be a good time now to act according to it

Reply:
lets postpone it to next releases https://github.com/MerginMaps/mobile/issues/2969